---
title: "Evolinc"
collection: research
type: "Research"
permalink: /research/evolinc
venue: "CyVerse"
location: "Tucson, Arizona, US"
---

Evolinc-I is a long intergenic noncoding RNA (lincRNA) identification workflow that also facilitates genome browser visualization of identified lincRNAs and downstream differential gene expression analysis 

<p/><img src='/images/Evolinc_logo_small_copy.png' alt="Evolinc" width="75" height="75" align="middle" hspace="20"> <b>Evolutionary analysis of Long non-coding RNA</b>

<p>
Evolinc is a comparative transcriptomics and genomics pipeline for quickly identifying sequence conserved lincRNAs for functional analysis. 
Evolinc is a two-part workflow (Evolinc-I and Evolinc-II) to identify lncRNAs from an assembled transcriptome file (GTF output from Cuffmerge/Cuffcompare) and then determine the extent to which those lncRNAs are conserved in the genome and transcriptome of other species 
</p>
<p>
  <b>Evolinc-I: lncRNA identification</b>
<p>
There are billions upon billions of RNA-Seq reads publicly available from which lncRNAs can be identified and more are being generated daily. Thus, what is needed are the computational resources to map and assemble these reads, and then a workflow to reproducibly identify the lncRNAs from these transcripts. Evolinc-I, working within the Discovery Environment (DE), is the perfect solution to identifying lncRNAs in your species of interest.</p>
</p>
<p>
  <b>Evolinc-II: Comparative genomic and transcriptomic analysis of lincRNAs</b>
  <p>
Your species of interest may have an overwhelmingly large dataset of uncharacterized lincRNAs. Determining where to start in your hunt for an interesting lincRNA can be difficult. One aspect examined with protein-coding genes is depth of conservation. LincRNA loci are typically poorly conserved. We propose that we can use the poor sequence conservation of lincRNA loci in general to filter large pools of lincRNAs down to a reasonable set of candidates for functional analysis. Depth of conservation, at the sequence level and at the transcriptional level, is the metric by which Evolinc-II helps you determine which lincRNAs to functionally characterize. With the understanding that conservation does not have to imply function, we think that this is the simplest and quickest way to identify target lincRNAs.</p>
</p>
</p>
  <b>Resources</b>
  <ol>
    <li><a href="https://wiki.cyverse.org/wiki/display/TUT/Evolinc+in+the+Discovery+Environment">CyVerse wiki describing the methodology and the use in DE</a></li>
    <li><a href="https://github.com/Evolinc/Evolinc-I">Evolinc-I Github page</a></li>
    <li><a href="https://github.com/Evolinc/Evolinc-II">Evolinc-II Github page</a></li>
    <li><a href="https://hub.docker.com/r/evolinc/evolinc-i">Evolinc-I Dokerhub image</a></li>
    <li><a href="https://hub.docker.com/r/evolinc/evolinc-ii">Evolinc-II Dokerhub image</a></li>
    <li><a href="http://journal.frontiersin.org/article/10.3389/fgene.2017.00052/abstract">Evolinc Publication in Frontiers in Genetics</a></li>
</ol>
</p>
